app/views/shared/_post_stats.mobile.haml

```diff
@@ -0,0 +1,19 @@
+.comment_container
+  .post_stats
+    - if @post.public?
+      %span.reshare_count
+        = @post.reshares.size
+
+    %span.comment_count
+      = @post.comments.size
+    %span.like_count
+      = @post.likes.size
+
+  %ul.comments
+    = render :partial => 'comments/comment', :collection => @post.comments, :locals => {:post => @post}
+
+    %li.comment.add_comment_bottom_link_container
+      = link_to "#", :class => "show_comments bottom_collapse active" do
+        = image_tag 'icons/arrow_up_small.png'
```

Comment:
While touching this code please update it to our new [styleguide](https://wiki.diasporafoundation.org/Styleguide) by preferring double over single quotes.